modules/ximgproc/src/structured_edge_detection.cpp

```diff
@@ -419,7 +419,16 @@ class StructuredEdgeDetectionImpl : public StructuredEdgeDetection
     /*!
      * This constructor loads __rf model from filename
      *
-     * \param filename : name of the file where the model is stored
+     * Refer to <https://github.com/opencv/opencv_extra/blob/master/testdata/cv/ximgproc/model.yml.gz>
```

Comment:
Ok, but please add before the declaration of the old create function `CV_DEPRECATED` specifier.